app/assets/scss/_save-search-page.scss

```diff
@@ -0,0 +1,26 @@
+.save-search-page {
+    .search-results-summary{
+        -webkit-box-sizing: border-box;
+        -moz-box-sizing: border-box;
+        box-sizing: border-box;
+        clear: both;
+        border-left: 9px solid $border-colour;
+        padding: 0 0.78947em;
+        margin-bottom: 0.78947em;
```

Comment:
this comes from `.panel` govuk scss toolkit. Need to look into possibility of using this instead of copying CSS.